cardano-explorer-db/src/Explorer/DB/Query.hs

```diff
@@ -150,10 +151,10 @@ queryTotalSupply = do
 
     -- Unfortunately the 'sum_' operation above returns a 'PersistRational' so we need
     -- to unWibble it.
-    unWibble :: Maybe (Value (Maybe Double)) -> Word64
+    unWibble :: Maybe (Value (Maybe Micro)) -> Micro
```

Comment:
This is just a copy/pasted verison of mine. However, mine has changed. It got a far better name, and was make into a top level function that is used twice in the same file.

Making this code long term maintainable should be an important consideration for any code we write.